e2e-test/resources/tsps/tsps_azure_set_up_control_workspace.py

```diff
@@ -90,6 +93,30 @@ def workspace_exists(billing_project_name, workspace_name, rawls_url, token):
     return response_json['workspace']['workspaceId']
 
 
+def add_user_to_billing_profile(rawls_url, billing_project_name, email_to_share, owner_token):
```

Comment:
i think this is already defined in the e2e test script.  Can we use that here or does the one in the e2e test not do what we want?

Reply:
yeah we can, I just need to refactor it into another helper script and I'm lazy. will do!